Vagrantfile

```diff
@@ -97,7 +97,7 @@ SCRIPT
 $kubeDashScript = <<SCRIPT
 
 # Kubernetes Dashboard Setup
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v1.10.1/src/deploy/recommended/kubernetes-dashboard.yaml
```

Comment:
Could you please make that into a (Makefile + Vagrantfile) variable as, e.g., seen on line 89 in this file?